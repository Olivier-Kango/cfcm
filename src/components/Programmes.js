import React from 'react';

const Programmes = () => (
  <div className="service">
    Nos Programmes
  </div>
);

export default Programmes;
