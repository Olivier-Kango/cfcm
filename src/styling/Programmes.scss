.programmes {
  padding: 5rem 4rem;
}
