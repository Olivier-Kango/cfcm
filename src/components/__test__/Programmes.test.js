import { render, screen } from '@testing-library/react';
import Programmes from '../Programmes';

describe('Testing the Services page', () => {
  test('should render', () => {
    render(<Programmes />);
    const about = screen.getByText(/Nos Programmes/i);
    expect(about).toMatchSnapshot();
  });
});
