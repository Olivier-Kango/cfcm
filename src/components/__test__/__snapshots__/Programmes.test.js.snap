// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Services page should render 1`] = `
<h2>
  Nos Programmes
</h2>
`;
